### PR TITLE
#4 Footnotes

### PR DESCRIPTION
**Pull request by JayDeeDee**
- fix styling of footnotes

### CODE REVIEW

File: css/area_footnotes.less [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ */
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes section  */
+.dokuwiki div.footnotes {
+  div.fn {
+    margin-bottom: .2rem;
+    display: table;//
+    vertical-align: top;//
```
Comment (splitbrain) [inline css/area_footnotes.less]: What's up with these trailing comments?

File: css/area_footnotes.less [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    .content { //
+      vertical-align: top;
+      line-height: 135%;
+    }
+
+    > sup {
+
+    }
```
Comment (splitbrain) [inline css/area_footnotes.less]: Why is this empty? Shouldn't this be the only `<sup>` formatted as a table-cell? there might be regular `<sup>`s in the footnote content.

File: css/area_footnotes.less [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes in content  */
+sup,
+.main-content sup,
+.main-content > div > p sup,
+.main-content > div > ul > li .li sup
```
Comment (splitbrain) [inline css/area_footnotes.less]: Why do we need so many selectors here?

File: css/area_footnotes.less (hunk 1 above)
Comment (JayDeeDee) [reply css/area_footnotes.less]: > [09dd0a6](https://github.com/cosmocode/dokuwiki-template-sprintdoc/commit/09dd0a60f884baad73f4a886073f20c16e494520) seems to be completely unrelated and bloats up this PR unnecessarily

reverted (will be part of separate merge request)

File: css/area_footnotes.less (hunk 2 above)
Comment (JayDeeDee) [reply css/area_footnotes.less]: good point, is done

File: css/area_footnotes.less (hunk 3 above)
Comment (JayDeeDee) [reply css/area_footnotes.less]: unfortunately these nested selectors are necessary 
(overwrite "fonts-size: inherit" of link elements in main-content without changing default font-size of sup)

**Pull request merged**